content/applications/inventory_and_mrp/manufacturing/management/split_merge.rst

```diff
@@ -0,0 +1,82 @@
+====================================
+Split and merge manufacturing orders
+====================================
+
+In Odoo *Manufacturing*, it is possible to create manufacturing orders for a single unit of an item
+or multiple units of the same item. In some cases, it may be necessary to split a manufacturing
```

Comment:
```suggestion
In Odoo *Manufacturing*, it is possible to create manufacturing orders for either a single unit or
multiple units of the same item. In some cases, it may be necessary to split a manufacturing
```
Just a slight, optional rewording